tests/rbd/rbd_peristent_writeback_cache.py

```diff
@@ -274,3 +276,16 @@ def device_cleanup(rbd):
             )
             if out_2:
                 log.error(f"RBD unmap failed for {device_name} ")
+
+
+def kill_fio(rbd):
+    """
+    Kill the fio process running on the client
+    """
+    time.sleep(60)
+    cmd = "ps -ef | grep fio"
+    out = rbd.exec_cmd(cmd=cmd, sudo=True, output=True)
+    if out and "fio --name=" in out:
```

Comment:
In my recent PR, I have changed this logic, because this was not working always. So can you please refer to my latest PR https://github.com/red-hat-storage/cephci/pull/3151/files for proper changes.

Reply:
let's discuss on this which to merge then.

Reply:
done